### DISCUSSION TITLE
Can any of the newer Roomba's be hacked like the old 780?

### DISCUSSION THREAD

**Discussion by SeguraUT** [iRobotEducation/create3_docs#371] [Questions]
We are looking to do some research on the Roombas. Some older Roombas, like the 780, could be hacked and modified for reach and testing. See the link below:

[https://hackaday.com/2015/12/05/hacklet-87-roomba-projects/]

We do not want to use the Create 3 because it lacks any vacuuming capability. Does anyone know if any of the current ones can be hacked? 

Thank you!

**Comment by kwujciak** [comment]
I know the Create® 3 robot does not allow hacking for modification, so I would assume the newer Roombas do not either, although I am not certain.
